substrate/runtime/system/src/lib.rs

```diff
@@ -69,6 +69,7 @@ pub trait Trait: Eq + Clone {
 	type Hashing: Hashing<Output = Self::Hash>;
 	type Digest: Parameter + Member + Default + traits::Digest;
 	type AccountId: Parameter + Member + MaybeDisplay + Ord + Default;
+	type Address: Parameter + Member + MaybeDisplay;
```

Comment:
Not super keen on having `Address` up here since it shouldn't be used prior to the staking module. What's the reason?

Reply:
for my reasoning see https://github.com/paritytech/polkadot/issues/212#issuecomment-402725936. where should we put `Address` instead?

Reply:
i was on the wrong track when i did that. reverted adding the `Address` https://github.com/paritytech/polkadot/pull/287/commits/2e3e4a89e46135f2c7dd2f39411c92025bb42e9e and  properly adapted to changed `Checkable` trait https://github.com/paritytech/polkadot/pull/287/commits/6e550d889fbbd2104368443d3251bdc639c95a46